org.lflang.ui/src/org/lflang/ui/LFUiModuleImpl.java

```diff
@@ -0,0 +1,317 @@
+package org.lflang.ui;
+
+import com.google.common.base.Objects;
+import com.google.inject.Provider;
+import java.io.PrintStream;
+import java.util.LinkedList;
+import java.util.List;
+import org.eclipse.jface.text.BadLocationException;
+import org.eclipse.jface.text.DocumentCommand;
+import org.eclipse.jface.text.IDocument;
+import org.eclipse.jface.text.IRegion;
+import org.eclipse.jface.text.TextUtilities;
+import org.eclipse.ui.console.ConsolePlugin;
+import org.eclipse.ui.console.IConsole;
+import org.eclipse.ui.console.MessageConsole;
+import org.eclipse.ui.console.MessageConsoleStream;
+import org.eclipse.ui.plugin.AbstractUIPlugin;
+import org.eclipse.xtend.lib.annotations.FinalFieldsConstructor;
+import org.eclipse.xtext.resource.containers.IAllContainersState;
+import org.eclipse.xtext.ui.editor.autoedit.AbstractEditStrategyProvider;
+import org.eclipse.xtext.ui.editor.autoedit.AbstractTerminalsEditStrategy;
+import org.eclipse.xtext.ui.editor.autoedit.CommandInfo;
+import org.eclipse.xtext.ui.editor.autoedit.DefaultAutoEditStrategyProvider;
+import org.eclipse.xtext.ui.editor.autoedit.MultiLineTerminalsEditStrategy;
+import org.eclipse.xtext.ui.editor.autoedit.SingleLineTerminalsStrategy;
+import org.eclipse.xtext.ui.shared.Access;
+import org.eclipse.xtext.xbase.lib.CollectionLiterals;
```

Comment:
I hadn't had the chance to review this PR, but I would recommend to steer away from the `xtext.xbase` dependencies like @petervdonovan suggested in his review of https://github.com/lf-lang/lingua-franca/pull/895.

Reply:
[This PR](https://github.com/lf-lang/lingua-franca/pull/914) should address this issue.